src/types/onyx/SearchResults.ts

```diff
@@ -204,7 +207,7 @@ type SearchTransaction = {
     transactionThreadReportID: string;
 
     /** The action that can be performed for the transaction */
-    action: string;
+    action: SearchTransactionAction;
```

Comment:
Where do we set this value? I found that the action is shown as `View` for a `Hold` expense


https://github.com/Expensify/App/assets/117511920/d76c4c09-f17f-4777-a2c9-5ecfdc6a8d8d

Reply:
This comes directly from backend via Onyx, we do not set it in any way. In your specific case if this action should be `Unhold` instead of view, then that would have to be handled via backend - and perhaps backend is not yet ready for this.

Btw: this is the place where data from api from onyx is decorated with some extra properties for the list, then passed down: https://github.com/Expensify/App/blob/main/src/libs/SearchUtils.ts#L137
CC @luacmartins

Reply:
Yea, the backend for hold/unhold is not ready yet so we'll have to test with dummy data, either manually updating it in Onyx or hardcoding it in the function @Kicu mentioned above.